README.md

```diff
@@ -107,7 +106,7 @@ accelerate launch scripts/finetune.py examples/openllama-3b/lora.yml \
 
   3. Install torch
   ```bash
-  pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
+  pip3 install torch==2.0.1+cu118 --index-url https://download.pytorch.org/whl/cu118
```

Comment:
```suggestion
  pip3 install -U torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
```

I think you do not need to specify the version of torch as latest is 2.0.1 one. Setting that index url also sets the cuda version. The thing I forgot to add was the `-U`

Reply:
thanks, I've updated it with the `-U` and removed the explicit version. I'm going to keep the removal of torchvision and torchaudio since those aren't needed I don't think